src/StaticRenderRegeneration/utils.ts

```diff
@@ -1,42 +1,42 @@
 export function getUrl(request: Request) {
-    try {
-        return new URL(request.url)
-    } catch {
-        return {
-            pathname: request.url,
-            search: request.url
-        }
-    }
+  try {
+    return new URL(request.url);
+  } catch {
+    return {
+      pathname: request.url,
+      search: request.url,
+    };
+  }
 }
 
 export function readCache(cacheFilePath: string): Uint8Array | null {
-    try {
-        return Deno.readFileSync(cacheFilePath)
-    } catch {
-        return null
-    }
+  try {
+    return Deno.readFileSync(cacheFilePath);
+  } catch {
+    return null;
+  }
 }
 
 // https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/digest
 export async function generateUUID(message: string): Promise<string> {
```

Comment:
Is the function name a UUID?
Maybe digest or hash is correct?

Reply:
Done